codecept.conf.js

```diff
@@ -1,7 +1,9 @@
 const { setHeadlessWhen, setCommonPlugins } = require('@codeceptjs/configure');
 // turn on headless mode when running with HEADLESS=true environment variable
 // export HEADLESS=true && npx codeceptjs run
-setHeadlessWhen(process.env.HEADLESS);
+setHeadlessWhen(process.env.HEADLESS || true);
```

Comment:
Will this always evalutate to true?

Reply:
Yes you're right. I fixed it in the most recent commit.